### PR TITLE
displayed DE table in the UI

### PR DESCRIPTION
**Pull request by airenzp**
## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/singleCellPlot.js [hunk 1]
```diff
@@ -144,26 +150,48 @@ class singleCellPlot {
...
+
+				const categoryName = this.dom.deselect.node().value.split(' ')?.[1]
+				if (!categoryName) return
+
+				const columnName = state.termdbConfig.queries.singleCell.DEgenes.columnName
+				const sample =
+					this.state.config.experimentID || this.state.config.sample || this.samples[0]?.experiments[0]?.experimentID
+				const args = { genome: state.genome, dslabel: state.dslabel, categoryName, sample, columnName }
+				const result = await dofetch3('termdb/singlecellDEgenes', { body: args })
+				const columns = [{ label: 'Gene' }, { label: 'Log2FC' }, { label: 'P-value' }]
```
Comment (xzhou82) [inline client/plots/singleCellPlot.js:177]: please label 3rd column as Adjusted P-value

**Comment by xzhou82** [comment]
thanks. after selecting a different sample, should clear the gene table

**Comment by airenzp** [comment]
> thanks. after selecting a different sample, should clear the gene table

Done

File: client/plots/singleCellPlot.js (hunk 1 above)
Comment (airenzp) [reply client/plots/singleCellPlot.js]: Done

**Review by xzhou82 — APPROVED** [review]
thanks. would be nice to be able sort genes by fold change or pvalue
will ask Robin to look at it from the point of gsea

File: client/plots/singleCellPlot.js [hunk 2]
```diff
@@ -144,26 +150,49 @@ class singleCellPlot {
...
+				DETableDiv.selectAll('*').remove()
+
+				const categoryName = this.dom.deselect.node().value.split(' ')?.[1]
+				if (!categoryName) return
+
+				const columnName = state.termdbConfig.queries.singleCell.DEgenes.columnName
+				const sample =
+					this.state.config.experimentID || this.state.config.sample || this.samples[0]?.experiments[0]?.experimentID
+				const args = { genome: state.genome, dslabel: state.dslabel, categoryName, sample, columnName }
+				const result = await dofetch3('termdb/singlecellDEgenes', { body: args })
```
Comment (xzhou82) [inline client/plots/singleCellPlot.js:177]: need error handling. on any err, do not render table

**Pull request merged**